common/app/conf/switches/ABTestSwitches.scala

```diff
@@ -172,9 +172,9 @@ trait ABTestSwitches {
 
   Switch(
     ABTests,
-    "ab-acquisitions-epic-design-variations",
+    "ab-acquisitions-epic-design-variations-v2",
     "Test design variations to the Epic",
-    owners = Seq(Owner.withGithub("Mullefa")),
+    owners = Seq(Owner.withGithub("jranks123")),
     safeState = Off,
     sellByDate = new LocalDate(2017, 3, 17),
```

Comment:
JS test expiries this Friday, is this by design (since test expiries don't have to match their Switch), or does it need to be extended (like the Switch)?